test/parallel/test-v8-serdes.js

```diff
@@ -118,3 +118,11 @@ const objects = [
   assert.deepStrictEqual(buf, ser.releaseBuffer());
   assert.strictEqual(des.getWireFormatVersion(), 0x0d);
 }
+
+{
+  // Unaligned Uint16Array read, with padding in the underlying array buffer.
+  const buf = Buffer.from('0'.repeat(64) + 'ff0d5c0404addeefbe', 'hex')
+      .slice(32);
```

Comment:
Is the idea here to create a buffer with a byteOffset > 0?  An IMO more reliable and future-proof way is this:
```js
let buf = Buffer.alloc(32 + 9);  // Hope I counted the bytes right!
buf.write('0'.repeat(64) + 'ff0d5c0404addeefbe', 'hex');
buf = buf.slice(32);
assert.strictEqual(buf.byteOffset, 32);  // Deterministic byte offset, unlike Buffer.from().slice()
// ...
```